core/core-impl/src/test/java/ai/timefold/solver/core/config/solver/EnvironmentModeTest.java

```diff
@@ -88,17 +87,16 @@ void determinism(EnvironmentMode environmentMode) {
         Solver<TestdataSolution> solver2 = SolverFactory.<TestdataSolution> create(solverConfig).buildSolver();
 
         switch (environmentMode) {
-            case NON_REPRODUCIBLE:
+            case NON_REPRODUCIBLE -> {
```

Comment:
I like we're doing this!

Just please double check the entire codebase for switching over environment modes and apply this consistently. (If you haven't already.)